src/uniparc/components/entry/UniParcFeaturesView.tsx

```diff
@@ -26,7 +25,7 @@ const convertData = (data: SequenceFeature[]): UniParcProcessedFeature[] =>
   sortBy(
     data.flatMap((feature) =>
       feature.locations.map((locationFeature) => ({
-        type: FeatureType.OTHER,
+        type: 'Other',
```

Comment:
About these changes, is there no drawback to changing from string enums to `string|string|string` at all?

Reply:
It might be easier to get cases like these to end up being typed as `string` if we're not strict enough with types, but here in this example we need to return a `UniParcProcessedFeature` whose `type` field is correctly typed so it would have caught the issue.

Reply:
Ok, so as long as we don't just use `string` on return types, it should be fine?

Reply:
It should, and as long as the components using this kind of data declare that they want these types, and not just plain strings.